beacon-chain/rpc/eth/beacon/handlers_validator.go

```diff
@@ -97,7 +97,10 @@ func (s *Server) GetValidators(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	epoch := slots.ToEpoch(st.Slot())
-	allBalances := st.Balances()
+	getBalance := func(idx primitives.ValidatorIndex) uint64 {
+		balance, _ := st.BalanceAtIndex(idx)
```

Comment:
We should handle the error here, if we retrieve a non-existent index then the caller should be made aware of it rather than assume that it is actually valid and `0`